Source/monstdat.h

```diff
@@ -106,24 +106,27 @@ struct MonsterData {
 	uint16_t hitPointsMinimum;
 	uint16_t hitPointsMaximum;
 	_mai_id ai;
-	/** Using @p monster_flag as bitflags */
+	/**
+	 * @brief Denotes monster's abilities defined in @p monster_flag as bitflags

```

Comment:
I don't understand `@p`, syntax error, brain out of context :shrug:

Reply:
This '@p` is to make `monster_flag` appear in typewriter font, since it is an enum

Reply:
ok